### PR TITLE
Sync everything to persistent storage to avoid writeback affects perf tests

### PR DESCRIPTION
**Pull request by azat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docker/test/performance-comparison/perf.py [hunk 1]
```diff
@@ -295,6 +295,9 @@ def do_create(connection, index, queries):
 
     reportStageEnd("create")
 
+# Let's sync the data to avoid writeback affects performance
+os.system('sync')
```
Comment (nickitat) [inline docker/test/performance-comparison/perf.py:299]: perf tests run for an hour at least, so it is questionable if a single sync in the beginning could help

Comment (azat) [reply docker/test/performance-comparison/perf.py:299]: It runs each test individually. So sync will be done after each group of CREATE queries

Comment (nickitat) [reply docker/test/performance-comparison/perf.py:299]: got it, then looks good. interesting to see how big will be the overhead

**Comment by azat** [comment]
Almost zero difference, but I still think that this is a good patch overall, since it may fix some spikes.

**Pull request merged**